vendor/assets/stylesheets/components/_carousel.scss

```diff
@@ -0,0 +1,34 @@
+.carousel {
+  overflow: hidden;
```

Comment:
Properties should be ordered height, overflow, perspective, position, transform-origin, transform-style, width